advanced_functionality/autogluon-multimodal-containers/scripts/multimodal_serve.py

```diff
@@ -0,0 +1,53 @@
+import base64
+import hashlib
+import json
+from io import BytesIO, StringIO
+
+import pandas as pd
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def _decode_and_save_base85_image(image_b85):
+    """saves Base65 encoded image to disk and returns unique name"""
+
+    image_bytes = base64.b85decode(image_b85)
+    image_hash = hashlib.md5(image_bytes).hexdigest()
```

Comment:
I like this. Better than my stupid way haha

Reply:
thanks!